themes/default/theme/webpack.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = function (env, { mode }) {
         output: {
             filename: "[name]-registry.js",
             chunkFilename: "chunk-[id]-registry.js",
-            path: `${process.cwd()}/../assets/js`,
+            path: `${process.cwd()}/../assets/registry/js`,
```

Comment:
when the build runs, will it produce this under the `/registry` route though, or will it be under the root, i.e. `/assets/registry/js`? I'm guessing you tested it out locally though.

Reply:
They'll end up at `/registry/css` and `/registry/js`, yeah. I did test locally so it _should_ be okay, but we'll run a test deployment to make sure nothing funny happens at the CDN level.